packages/runtime-dom/src/components/fallthroughAttrs.ts

```diff
@@ -0,0 +1,12 @@
+import { Data } from '@vue/runtime-core/src/component'
```

Comment:
Avoid directly importing from files in another package. This can cause issues in generated `d.ts` files. Just use `Record<string, any>` here.